docs/administer/Idp_groups_integration.md

```diff
@@ -0,0 +1,142 @@
+
+To use IdP groups in Percona Everest RBAC, set up the groups claim in your IdP provider settings. Here are the steps to configure the groups claim:
+{.power-number}
+
+1. Navigate to **Security > API**.
+
+2. In the list, find the API server that you use and click **Edit**.
+
+3. Click **Scopes > Add Scope**. 
+
+    !!! note
+        A scope specifies the level and type of access a client application can request from an Identity Provider (IdP) during the authentication process.
+      
+    Add a scope called **groups** while keeping all other options to their default settings.
+
+    ![!image](../images/scr_scope.png)
+
+4. Click **Claims > Add Claim**. 
+
+    !!! note
+        A claim specifies the scope to activate and the information required within that scope.
+
+    1. Add a claim named **groups**.
+
+    2. Set the **Include in token type** to **ID Token** and choose **Always**.
+
+    3. Change the **Value type** to **Groups**.
+
+    4. Select the **Filter** as **Matches regexp** and add a filter to match the desired Okta groups for Everest, for example: .*.
+
+    5. Set **Include in** to **groups**, which is the scope you created earlier.
+
+    ![!image](../images/scr_claim.png)
+
+5. Click **Create**.
+
+6. Click **Access Policies > Add Policy**. A policy restricts how this authorization server is used.
+
+    1. Add **Name** and **Description**.
+
+    2. Assign the policy to the client (Application Integration) you created earlier. The field should auto-complete as you type. Create the policy.
+
+    3. Create the policy.
+
+    ![!image](../images/scr_policy.png)
+
+6. Add a rule to the policy. A Rule defines the details of usage:
+
+    1. Add a name. If you prefer, you can use **default** option.
+
+    2. Fine-tune the settings to align with your organization's security policy. Here are some suggestions:
+
+        - Uncheck all grant types except for the **Authorization Code**.
+        -  Adjust the **token lifetime** to determine how long a session can remain active.
+        - Restrict the refresh **token lifetime** or consider deactivating it completely.
+
+    ![!image](../images/scr_rule.png)
+
+7. Click **Create Rule**.
+
+
+## Configure Group claims in OKTA
+
+During the SSO authentication, if the group's scope is requested but the Identity Provider (IdP), like Okta, lacks the necessary groups claim configuration or does not support it by default, SSO could fail or not function as expected.
+
+
+To retrieve the OIDC groups, you need to include the scope by specifying the following fields:
+
+`--scopes openid,profile,email,groups`
```

Comment:
@recharte should we specify what is the command where this option should be used? 
Smth like 

by configuring OIDC, add the `--scopes` parameter:
```
everestctl settings oidc configure --issuer-url https://example.com --client-id 123456 --scopes openid,profile,email,groups
```

Reply:
Yes, funny enough I left the exact same comment while reviewing the RNs: https://github.com/percona/everest-doc/pull/204#discussion_r1953195879

Reply:
Fixed